### ISSUE TITLE
Screen reader is not announcing the tooltip error message information when focus moves to the date control in Windows.

### ISSUE THREAD

**Issue by msftedad** [ceolter/ag-grid#9947]
### Link to reproducible scenario

Typescript Example - Filter Date - Date Filter

### Describe the bug

Steps to reproduce 

1. Open URL: [Typescript Example - Filter Date - Date Filter](https://www.ag-grid.com/examples/filter-date/date-filter/typescript/) in the latest edge browser.
2. Turn on the screen reader.
3. Press tab key to move to date control and enter the invalid input value in the date.
4. Verify whether screen reader is announcing the tooltip error message information when focus moves to the date control or not.

Actual behaviour
Screen reader is not announcing the tooltip error message information when focus moves to the date control in Windows. 
Narrator behavior: It is announcing it as 'year date filter input spin button 9876 minimum 2000 and maximum 2021'.
NVDA behavior: It is announcing it as 'Year Date Filter Input  spin button  9876'.

https://github.com/user-attachments/assets/8c1bd765-f7da-46a9-9124-7d2a6e953305
<img width="972" alt="Image" src="https://github.com/user-attachments/assets/5d105778-92b1-4565-bd5d-52060ccf7ce1" />

Expected behaviour
Screen reader should announce the tooltip error message information when focus moves to the date control in Windows. It should announce it as value must be greater than 01/01/2000 or later tool tip information

More information
Issue is observed with all three screen readers.

### Version

31.3.4

### Does the issue occur for a specific framework only?

All languages

### Is the issue only observable on a specific browser?

All browsers

**Comment by AG-Zoheil** [comment]
Hello 

Thank you for the suggestion.

Currently there's no way to implement this behaviour with AG Grid. However, we agree this would be a valuable addition to the product and we have added a feature request for this.

We have added this requirement to our backlog and we are tracking it with the following reference: **AG-14234**

We have no timeline for this at the moment, but this can still be picked up by our team when they're making modifications to this functional area of AG Grid. Please use the functionality currently provided by AG Grid until this is delivered at a later point.

You can follow the status of this item on the product pipeline page::
https://ag-grid.com/pipeline/

The best way to track this is to sign up for AG Grid new release notifications using [the instructions here](https://ag-grid.zendesk.com/hc/en-us/articles/360002648058-How-do-I-sign-up-for-ag-Grid-release-notifications-). This way you'll know as soon as a new version is out and you can check whether this specific item was implemented on [the changelog page](https://www.ag-grid.com/ag-grid-changelog/).

Thanks again for bringing this up with us.

Kind regards,
Zoheil
